x/margin/keeper/genesis_test.go

```diff
@@ -0,0 +1,26 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/Sifchain/sifnode/x/margin/test"
+)
+
+func TestKeeperExportGenesis(t *testing.T) {
+	ctx, app := test.CreateTestAppMargin(false)
+	marginKeeper := app.MarginKeeper
+	assert.NotNil(t, marginKeeper)
+	state := marginKeeper.ExportGenesis(ctx)
+	assert.NotNil(t, state)
```

Comment:
We should test for what we want to export here, params and mtps for example.

Reply:
updated the test from init/export genesis functions so it asserts the existence of the params field along with its fields but I am not familiar with MTP-related fields with the GenesisState struct, can you clarify thanks @timlind 

fyi: https://github.com/Sifchain/sifnode/pull/2242/commits/e1ec8f1c00602bdb09806d687c20e6d047416387#diff-6f2afbfb379f05d7031c6f9a0d9754f691a5a890818e5a367beaa7c255c640c9R64-R87